.github/validation_samples/kaon_enhanced/config.py

```diff
@@ -0,0 +1,158 @@
+import os
+import sys
+
+from LDMX.Framework import ldmxcfg
+
+thisPassName = 'test'
+p=ldmxcfg.Process(thisPassName)
+
+p.termLogLevel = 0
+# This need to be set but has no meaning given p.totalEvents
+p.maxEvents = 1
+# kaon-focused sample takes about twice as long as normal PN,
+# so we ask for half as many events such that validation jobs stay
+# time-limited by the PN
+p.totalEvents = int(os.environ['LDMX_NUM_EVENTS']) // 2
+p.run = int(os.environ['LDMX_RUN_NUMBER'])
+
+from LDMX.SimCore import generators as gen
+from LDMX.SimCore import bias_operators
+from LDMX.SimCore import kaon_physics
+from LDMX.SimCore import photonuclear_models as pn
+from LDMX.Biasing import ecal
+from LDMX.Biasing import filters
+from LDMX.Biasing import particle_filter
+from LDMX.Biasing import util
+from LDMX.Biasing import include as includeBiasing
+
+detector = 'ldmx-det-v14-8gev'
+generator=gen.single_8gev_e_upstream_tagger()
+bias_factor = 550.
+bias_treshold = 5000.
+
+mySim = ecal.photo_nuclear(detector, generator)
+mySim.description = f'8 GeV ECal Kaon PN simulation, xsec bias {bias_factor}' 
+mySim.biasing_operators = [ bias_operators.PhotoNuclear('ecal',bias_factor,bias_treshold,only_children_of_primary = True) ]
+
+# Configure the sequence in which user actions should be called.
+includeBiasing.library()
+mySim.actions.clear()
+mySim.actions.extend([
+        filters.TaggerVetoFilter(thresh=2*3800.),
+        # Only consider events where a hard brem occurs
+        filters.TargetBremFilter(recoil_max_p = 2*1500.,brem_min_e = 2*2500.),
+        # Only consider events where a PN reaction happnes in the ECal
+        filters.EcalProcessFilter(),
+        # Tag all photo-nuclear tracks to persist them to the event.
+        util.TrackProcessFilter.photo_nuclear()
+])
+
+# set up upKaon parameters
```

Comment:
I guess this also would be worth a comment, I'm guessing that the name upKaons doesn't immediately tell a new user what we are doing.

Reply:
Any suggestions for the text to use?

Reply:
Sure, we can steal from the docs directly :) 

Reduces the charged kaon lifetimes by a factor 1/50 and forces decays to be into one of the leptonic decay modes.

Reply:
Sure, done now!